src/components/confirm/confirm.d.ts

```diff
@@ -2,14 +2,30 @@ import * as React from "react";
 import { DialogProps } from "../dialog/dialog";
 
 export interface ConfirmProps extends DialogProps {
-  /** Allows to setup buttonType into cancel button */
-  cancelButtonType?: "primary" | "secondary" | "tertiary";
-  // ** Customise the cancel button label */
+  /** Color variants for new business themes: "primary" | "secondary" | "tertiary" | "dashed" | "darkBackground" */
+  cancelButtonType?: "primary" | "secondary" | "tertiary" | "dashed" | "darkBackground";
+  /** Color variants for new business themes: "primary" | "secondary" | "tertiary" | "dashed" | "darkBackground" */
+  confirmButtonType?: "primary" | "secondary" | "tertiary" | "dashed" | "darkBackground";
+  /** Customise the cancel button label */
   cancelLabel?: string;
-  // ** Customise the confirm button label */
+  /** Customise the confirm button label */
   confirmLabel?: string;
   /** Apply destructive style to the buttons */
   destructive?: boolean;
+  /** Apply destructive style to the cancel button */
+  cancelButtonDestructive?: boolean;
+  /** Apply destructive style to the confirm button */
+  confirmButtonDestructive?: boolean;
+  /** Defines a cancel button Icon position related to the children: "before" | "after" */
+  cancelButtonIconPosition: "before" | "after";
```

Comment:
I think this prop and the three props below need a `?:` as they are not required.

Reply:
good point, thank you, changed